pom.xml

```diff
@@ -107,6 +147,208 @@
+                  </excludes>
+                </bannedDependencies>
+              </rules>
+            </configuration>
+          </execution>
+          <execution>
+            <id>ban-logging-frameworks</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <skip>${enforcer.skipBanLoggingFrameworks}</skip>
+              <rules>
+                <bannedDependencies>
+                  <message>Do not use and do exclude all transitive uses of logging frameworks.</message>
+                  <excludes>
+                    <exclude>org.slf4j:slf4j-simple</exclude>
+                    <exclude>commons-logging</exclude>
+                    <exclude>ch.qos.logback</exclude>
+                    <exclude>org.apache.logging.log4j</exclude>
+                    <exclude>org.apache.log4j</exclude>
+                    <exclude>log4j</exclude>
+                    <exclude>org.tinylog</exclude>
+                  </excludes>
+                  <includes>
+                    <include>org.slf4j:slf4j-simple:*:*:test</include>
+                  </includes>
+                </bannedDependencies>
+              </rules>
+            </configuration>
+          </execution>
+          <execution>
+            <id>enforce-scope-logger-bridges</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <skip>${enforcer.skipEnforceScopeLoggerBridges}</skip>
+              <rules>
+                <bannedDependencies>
+                  <message>Logger bridges must use scope runtime or test.</message>
+                  <excludes>
+                    <exclude>org.slf4j:jul-to-slf4j</exclude>
+                    <exclude>org.slf4j:jcl-over-slf4j</exclude>
+                    <exclude>org.slf4j:log4j-over-slf4j</exclude>
+                  </excludes>
+                  <includes>
+                    <include>org.slf4j:jul-to-slf4j:*:*:runtime</include>
+                    <include>org.slf4j:jul-to-slf4j:*:*:test</include>
+                    <include>org.slf4j:jcl-over-slf4j:*:*:runtime</include>
+                    <include>org.slf4j:jcl-over-slf4j:*:*:test</include>
+                    <include>org.slf4j:log4j-over-slf4j:*:*:runtime</include>
+                    <include>org.slf4j:log4j-over-slf4j:*:*:test</include>
+                  </includes>
+                </bannedDependencies>
+              </rules>
+            </configuration>
+          </execution>
+          <execution>
+            <id>enforce-scope-javaee</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <skip>${enforcer.skipEnforceScopeJavaEE}</skip>
+              <rules>
+                <bannedDependencies>
+                  <message>Use Java EE modules with scope provided or test.</message>
```

Comment:
(Minor)

Not sure whether this rule in the current form serves any benefit, when we are on JakartaEE, not JavaEE

Reply:
Do I remove the rule, or move it to Jakarta?
- If I need to move it, what dependencies should be enforced with what scope?

Reply:
I removed enforce JavaEE